SConstruct

```diff
@@ -90,7 +90,6 @@ if arch == "aarch64" or arch == "larch64":
     "/usr/lib",
     "/system/vendor/lib64",
     "/system/comma/usr/lib",
-    "#third_party/nanovg",
```

Comment:
let's keep this stuff for now to reduce the diff for this PR. you can make a follow up to cleanup all the references and delete nanovg from phonelibs